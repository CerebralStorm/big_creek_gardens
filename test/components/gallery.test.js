import React from 'react'
import Gallery from '../../app/javascript/components/gallery'

test('render a Gallery', () => {
  const wrapper = shallow(<Gallery />);
  expect(wrapper).toMatchSnapshot();
});