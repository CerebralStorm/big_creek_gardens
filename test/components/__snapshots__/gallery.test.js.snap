// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render a Gallery 1`] = `
<div>
  <h1>
    Gallery
  </h1>
</div>
`;
